homeassistant/components/habitica/coordinator.py

```diff
@@ -0,0 +1,67 @@
+"""Data Coordinator for Habitica."""
```

Comment:
Let's not move this yet.

Let's just keep it in here and directly migrate it to a proper update coordinator in a followup

Reply:
i put it back where it was